dev-resources/circleci_test/config.clj

```diff
@@ -0,0 +1,14 @@
+{:selectors {:es-store :es-store
```

Comment:
Something that can be adapted from the other project:

```clj
{:reporters [circleci.test.report/clojure-test-reporter
             user.test.reporter/time-reporter]}
```

(again, I'm leaving the suggestion in case it's easy, particularly if we opt to extract some code to a repo)

Reply:
Nice. I'll write one with namespace granularity later.